CHANGELOG/CHANGELOG-0.3.md

```diff
@@ -1,3 +1,15 @@
+## v0.3.2
+
+Changes since `v0.3.1`:
+
+### Bug or Regression
+
+- Add permission to update frameworkjob status. (#798, @k8s-infra-cherrypick-robot)
```

Comment:
```suggestion
- Add permission to update frameworkjob status. (#798, @tenzen-y)
```

I'm wondering why bot is mentioned 🤔

Reply:
probably because of automatic cherry-pick. Fixed.